packages/size-limit/get-config.js

```diff
@@ -109,6 +109,7 @@ module.exports = async function getConfig (plugins, process, args, pkg) {
     config.checks = await Promise.all(
       result.config.map(async check => {
         if (check.path) {
+          check.filePath = check.path
           check.path = await globby(check.path, { cwd: config.cwd })
```

Comment:
Can I ask you to refactor it a little. It was a mistake to use `path` for pattern and for the files.

Can you replace it to:

```js
          check.files = await globby(check.path, { cwd: config.cwd })
```

Reply:
I will try, but it will affect not only one place)